website/snippets/_new-sl-setup.md

```diff
@@ -3,17 +3,13 @@ You can set up the dbt Semantic Layer in dbt Cloud at the environment and projec
 - You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
     * Enterprise plan &mdash; Developer license with Account Admin permissions. Or Owner with a Developer license, assigned Project Creator, Database Admin, or Admin permissions.
     * Team plan &mdash; Owner with a Developer license.
+    * If you are using a free trial dbt Cloud account, you are on a trial of the Team plan as an Owner, so you're good to go.
 - You must have a successful run in your new environment.
```

Comment:
this reads like Prereqs. consider moving this info to step page 2. that way all the requirements are grouped together